app/__mocks__/@aries-framework/react-hooks.ts

```diff
@@ -23,6 +23,11 @@ const useAgent = () => ({
   agent: {
```

Comment:
These mocks were added for the unit tests I needed to remove. Will probably come in handy though, so i left them.